Sources/Engine/Brushes/BrushPolygon.cpp

```diff
@@ -509,7 +509,7 @@ CBrushPolygon &CBrushPolygon::CopyPolygon(CBrushPolygon &bp)
   bpo_boxBoundingBox=bp.bpo_boxBoundingBox;
   bpo_pbscSector=bp.bpo_pbscSector;
   bpo_rsOtherSideSectors.Clear();
-  bpo_lhShadingInfos;
+  //bpo_lhShadingInfos; // FIXME: DG: was this missing "= bp.bpo_lhShadingInfos" or something like that?
```

Comment:
well, /could/ be, but some other fields were cleared or set to NULL, so I don't know for sure.. and it seems to work like it is, so unless a bug appears due to that field not being copied, maybe it should just be left commented out?